code/game/mecha/combat/gorilla.dm

```diff
@@ -0,0 +1,198 @@
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/cannon
+	name = "8.8cm KwK 47"
+	desc = "<i>Precision German engineering!</i>" // Why would you ever take this off the mech, anyway?
+	icon_state = "mecha_uac2"
+	equip_cooldown = 55 // 5.5 seconds
+	projectile = /obj/item/projectile/bullet/cannon
+	fire_sound = 'sound/weapons/cannon.ogg'
+	projectiles = 1
+	projectile_energy_cost = 1000
+	salvageable = 0 // We don't want players ripping this off a dead mech. Could potentially be a prize for beating it if Devs bless me and someone offers a nerf idea.
+
+/obj/item/projectile/bullet/cannon
+	name ="armor-piercing shell"
+	icon = 'icons/obj/grenade.dmi'
+	icon_state = "shell"
+	damage = 1000 // In order to 1-hit any other mech and royally fuck anyone unfortunate enough to get in the way.
+
+	on_hit(var/atom/target, var/blocked = 0)
+		explosion(target, 0, 0, 2, 4)
+		return 1
+
+/* // GLITCHY UND LAGGY. Will later look into fixing.
+/obj/item/mecha_parts/mecha_equipment/weapon/ballistic/mg42
+	name = "Maschinengewehr 60"
+	icon_state = "mecha_uac2"
+	equip_cooldown = 10
+	projectile = /obj/item/projectile/bullet/midbullet2
+	fire_sound = 'sound/weapons/mg42.ogg'
+	projectiles = 1000
+	projectiles_per_shot = 5
+	deviation = 0.3
+	projectile_energy_cost = 20
+	fire_cooldown = 1
+	salvageable = 0 // We don't want players ripping this off a dead mech.
+*/
+
+/obj/effect/decal/mecha_wreckage/gorilla
+	name = "Gorilla wreckage"
+	desc = "... Blitzkrieg?"
+	icon = 'icons/mecha/mecha64x64.dmi'
+	icon_state = "pzrwreck"
+	layer = 4 // so it overlaps other people
+	pixel_x = -16
+	anchored = 1 // It's fucking huge. You aren't moving it.
+
+/obj/mecha/combat/gorilla
+	name = "Gorilla"
+	desc = "<b>Blitzkrieg!</b>" //stop using all caps in item descs i will fight you. its redundant with the bold.
+	icon = 'icons/mecha/mecha64x64.dmi'
+	icon_state = "pzrmech"
+	initial_icon = "pzrmech"
+	pixel_x = -16
+	step_in = 10
+	health = 5000
+	opacity = 0 // Because there's big tall legs to look through. Also it looks fucky if this is set to 1.
+	deflect_chance = 50
+	damage_absorption = list("brute"=0.1,"fire"=0.8,"bullet"=0.1,"laser"=0.6,"energy"=0.7,"bomb"=0.7) //values show how much damage will pass through, not how much will be absorbed.
+	max_temperature = 35000 //Just a bit better than the Durand.
+	infra_luminosity = 3
+	var/zoom = 0
+	var/smoke = 5
+	var/smoke_ready = 1
+	var/smoke_cooldown = 100
+	var/datum/effect/effect/system/smoke_spread/smoke_system = new
+	wreckage = /obj/effect/decal/mecha_wreckage/gorilla
+	add_req_access = 0
+	internal_damage_threshold = 25
+	force = 60
+	max_equip = 4
+	New()
```

Comment:
Relative pathing needs to be changed to absolute pathing.

Reply:
Will handle all the relative pathings too, thanks.